db_objects/service.class.php

```diff
@@ -499,8 +499,8 @@ function getInstancesData($params, $logic='OR', $order='')
 	public function replaceItemKeywords($title, $iteminfo)
 	{
 		$title = str_replace('%title%', $iteminfo['title'], $title);
-		$title = str_replace('%alt_title%', $iteminfo['alt_title'], $title);
-		$title = str_replace('%ccli_number%', $iteminfo['ccli_number'], $title);
+        $title = str_replace('%alt_title%', $iteminfo['alt_title'] ?? "", $title);
+        $title = str_replace('%ccli_number%', $iteminfo['ccli_number'] ?? "", $title);
```

Comment:
```suggestion
        $title = str_replace('%alt_title%', $iteminfo['alt_title'] ?? '', $title);
        $title = str_replace('%ccli_number%', $iteminfo['ccli_number'] ?? '', $title);
```

Feel free to ignore this suggestion, it just seems nicer to me to be consistent about quotes within the same line.